sdk/storage/storage-blob/README.md

```diff
@@ -93,18 +91,6 @@ Or
 const AzureStorageBlob = require("@azure/storage-blob");
 ```
 
-### JavaScript bundle
```

Comment:
I think it's necessary to document the bundle guideline when removing the default browser bundle from npm. Web developer beginners maybe not familiar with concepts like Node.js, npm, webpack, rollup or browserify. It's best if Azure JS SDK provides a unified guideline, and we can add links here.

Browser sample added below is not that very "highlight".

Reply:
Agree, will be working on such a document and let you know when its available, and will send PRs with links.